.gitlab/spack_packages/sundials/package.py

```diff
@@ -666,10 +698,9 @@ def initconfig_hardware_entries(self):
                     cmake_cache_path("HIP_PATH", spec["hip"].prefix),
                     cmake_cache_path("HIP_CLANG_INCLUDE_PATH", spec["llvm-amdgpu"].prefix.include),
                     cmake_cache_path("ROCM_PATH", spec["llvm-amdgpu"].prefix),
-                    cmake_cache_string("AMDGPU_TARGETS", spec.variants["amdgpu_target"].value)
+                    cmake_cache_string("AMDGPU_TARGETS", spec.variants["amdgpu_target"].value[0])
```

Comment:
Is `[0]` necessary or will this work with multiple `amdgpu_target` values?

Reply:
I'll check if it's necessary, though I recall earlier there were issues with parsing the target incorrectly resulting in an empty entry

Reply:
There is a fix to this, which is to do `",".join(spec.variants["amdgpu_target"].value`

Reply:
Should these be comma separated or semi-colon separated?

Reply:
I believe cmake will want semi-colon separated.

Reply:
The ROCm documentation had them as commas, but it might have been in a different context